Flat/src/GeneralTree.cc

```diff
@@ -885,6 +911,10 @@ void GeneralTree::WriteTree(TTree *t) {
     }
   }
 //ENDCUSTOMWRITE
+    if (photonic) {
```

Comment:
This has to go inside the `//STARTCUSTOMWRITE ... //ENDCUSTOMWRITE` block

Reply:
To clarify - the config builder is not guaranteed to preserve custom code like this unless it resides inside such blocks.

Reply:
Done. Indeed, I wrote the if statement just where the custom code put it. I modified it now